test/fixtures/cookbooks/test/recipes/default.rb

```diff
@@ -24,6 +24,7 @@
 
 if platform_family?('debian')
   apt_update
+  include_recipe 'build-essential'
```

Comment:
Yeah, please use ``build_essential`` here instead and remove the cookbook dep.

Reply:
remove the cookbook dep?

Reply:
I get it. There's a built in resource now.  Thanks